lib/units/unit.cpp

```diff
@@ -72,7 +74,7 @@ Unit &Unit::operator*=(const Unit &other) { return *this = *this * other; }
 
 Unit &Unit::operator/=(const Unit &other) { return *this = *this / other; }
 
-Unit &Unit::operator%=([[maybe_unused]] const Unit &other) { return *this; }
+Unit &Unit::operator%=(const Unit &) { return *this; }
```

Comment:
I think this needs to check for `none`.

Reply:
Do we want `m % None` to succeed but `m / None` to fail? (And reverse order of operands) Seems wrong given that all other binary ops are forbidden.